pulsetranslator/pulsetranslator.py

```diff
@@ -25,6 +25,9 @@
 
 
 class PulseBuildbotTranslator(object):
+    # Available buildbot results
+    # http://hg.mozilla.org/build/buildbot/file/08b7c51d2962/master/buildbot/status/builder.py#l25
+    results = ['success', 'warnings', 'failure', 'skipped', 'exception', 'retry']
```

Comment:
We actually have a discussion about the usefulness of the value over on https://bugzilla.mozilla.org/show_bug.cgi?id=1194207#c5. It looks like we always set Success.